plugins/item_tasks/web_client/views/TaskListView.js

```diff
@@ -1,31 +1,17 @@
 import View from 'girder/views/View';
-import PaginateWidget from 'girder/views/widgets/PaginateWidget';
+import router from 'girder/router';
 
-import ItemTaskCollection from '../collections/ItemTaskCollection';
-import template from '../templates/taskList.pug';
-import '../stylesheets/taskList.styl';
+import PaginateTasksWidget from './PaginateTasksWidget';
 
 var TaskListView = View.extend({
     initialize: function () {
-        this.collection = new ItemTaskCollection();
-
-        this.paginateWidget = new PaginateWidget({
-            collection: this.collection,
+        this.paginateWidget = new PaginateTasksWidget({
+            el: this.$el,
             parentView: this
+        }).once('g:selected', function (params) {
+            let taskId = params.taskId;
+            router.navigate('item_task/' + taskId + '/run', {trigger: true});
```

Comment:
This is a good case for using a template string

    `item_task/${taskId}/run`